DM-21317: Add technote_aastex template (#45)
lsst/templates: project_templates/technote_aastex/README.md

> # technote_latex

Reviewer: ```suggestion
# technote_aastex
```

## Template variables

### cookiecutter.org

The organization responsible for the document.

### cookiecutter.series

The identifier of the technote series which will be used ot name the repo
Choose the series that fits the document's purpose or aligns with the organization creating the document.
For now to make this simpler we assume Papers are DMTN or PSTN. 

- `DMTN` for Data Management technical notes. 
- `PSTN` for Project Science Team technical notes.

### cookiecutter.serial_number

The serial number. Use three digits padded with zeros.
If you are creating a technical note manually with this template, see the [Create a technote](https://developer.lsst.io/project-docs/technotes.html#create-a-technote) instructions for how to determine the serial number.

### cookiecutter.github_org

The GitHub organization where this technote resides.
Choose a GitHub organization that matches the [series](#cookiecutter_series):

- `lsst-dm` for the  DMTN series.
- `lsst-pst` for the PSTN series.

### cookiecutter.title

The title of the technote.

### cookiecutter.first_author

> The id  of the first author, the ids of authos may be foud in the 

Reviewer: ```suggestion
The ID of the first author, the ids of authors may be found in the 
```

The `acronyms.tex` file is not ignored so that `make acronyms.tex` does not need to be run for every document build.

## .travis.yml

Example: [.travis.yml](testn-000/.travis.yml).

The Travis CI configuration file.
You shouldn't have to modify this file unless you have a novel preprocessing build step (see below).

**Tip:** Travis CI provides a flexible Python environment.
It's likely easier to run preprocessing scripts directly from the Travis CI environment, rather than within the lsst-texmf Docker container:

- Install additional Python dependencies in the `install` phase of the `.travis.yml` file.
- Add bash command for the preprocessing steps to the `script` list of the `.travis.yml` file, before the `docker run` command.
- Structure your `Makefile` so that files built in advance in the Travis CI environment are automatically used as-is by the `docker run` command.

For more information about using the CI environment, see the [Travis CI documentation](https://docs.travis-ci.com).

## acronyms.tex

Example: [acronyms.tex](testn-000/acronyms.tex).

This file is generated and updated by the `make acronyms.tex` command.

> ## authors.tex

Reviewer: ```suggestion
## authors.tex

```

(add a blank line below)

See the [copyright](../copyright) template and [Copyrights for LSST DM work and the COPYRIGHT file](https://developer.lsst.io/legal/copyright-overview.html).

### LICENSE

Example: [LICENSE](testn-000/LICENSE).

Generally speaking, LSST documentation is licensed under CC-BY 4.0.
See [Licensing LSST DM source code and content](https://developer.lsst.io/legal/licensing-overview.html) in the Developer Guide for more information.

### local.bib

Example: [local.bib](testn-000/local.bib).

Add BibTeX citations to this file that aren't already available in [lsst-texmf](https://lsst-texmf.lsst.io) (the [lsstbib/](#testn-000/local_bib).
See the [Updating bibliographies](https://lsst-texmf.lsst.io/developer.html#updating-bibliographies) documentation in lsst-texmf for how to migrate local bibliography data upstream into [lsst-texmf](https://lsst-texmf.lsst.io).

### Makefile

Example: [Makefile](testn-000/Makefile).

The built-in targets are:

- `make`: compiles the PDF document by running `xelatex` and `bibtex` iteratively.
  This command is used by the CI environment that pushes the PDF to LSST the Docs (lsst.io).
  See the [.travis.yml](#travisyml) file.

> - `make acronyms.tex`: regenerates the [acronyms.tex](#acronymstex) file.

Reviewer: Add:

```
- `make authors.tex`: regenerates the [authors.tex](#authorstex) file.
```
